rosa_planning/rosa-cloud-expert-prereq-checklist.adoc

```diff
@@ -157,6 +157,16 @@ Prerequisites needed from a networking standpoint.
 
 * Configure your firewall to allow access to the domains and ports listed in xref:../rosa_planning/rosa-sts-aws-prereqs.adoc#osd-aws-privatelink-firewall-prerequisites_rosa-sts-aws-prereqs[AWS firewall prerequisites].
 
+=== Additional custom security groups
```

Comment:
Needs an ID above

Reply:
This is part of the ported mobb.ninja content, so it has different standards for now.